### ISSUE TITLE
Export map as vector image

### ISSUE THREAD

**Issue by sylvaincaillot** [Mapsui/Mapsui#2168]
Hi,

I am using Mapsui to generate maps that are later exported as images. By default, it is a PNG but I would like to export the map into a vector image such as EMF or SVG in order to edit it later if needed.

How could I do that?

**Comment by charlenni** [comment]
That isn't possible, because Mapsui handling only bitmap map tiles, no vector map tiles.

**Comment by sylvaincaillot** [comment]
Thank you for the quick reply. I thought that you would be quite useful to have a similar export tool as QGIS for vector files such as .emf or.svg.

**Comment by pauldendulk** [comment]
SkiaSharp can handle vectors as input but can only output bitmaps. If something like this would be implemented this should be implemented through a renderer. So, we would have to add a Mapsui.Rendering.Svg. In the past we had a renderer for WPF which was vector based, and I implemented a pdf renderer for another platform once. 

But even if we have something like this you have to keep in mind that if you use bitmaps as input they will also be represented as bitmaps in SVG/EMF/PDF, so, as charlenni mentioned, the bitmap tile layer sources will still be bitmaps. If would be nice to have vector tiles for that scenario, but you could also use another NTS based data source for this.

Summing up, this is not implemented, and this is not easy to implement, but is something to keep in mind for the long run (especially for us, the developers). This also means we need to keep SkiaSharp contained in the renderer.
